quesma/clickhouse/clickhouse.go

```diff
@@ -34,7 +34,8 @@ const (
 	// Above this number of columns we will use heuristic
 	// to decide if we should add new columns
 	alwaysAddColumnLimit  = 100
-	AlterColumnUpperLimit = 1000
+	alterColumnUpperLimit = 1000
+	fieldFrequency        = 10
```

Comment:
Arbitrarily chosen value, we can change that according to results of our empirical tests